asf_heat_pump_suitability/pipeline/prepare_features/off_gas.py

```diff
@@ -0,0 +1,18 @@
+from asf_heat_pump_suitability.getters.get_datasets import get_df_spa_offgasgrid
+import polars as pl
+
+
+def add_off_gas_feature(df: pl.DataFrame) -> pl.DataFrame:
```

Comment:
You could do the processing of the off-gas feature here and then add as a column in the script. It just means that if we wanted to load the transformed off-gas dataset without EPC for some reason at a later point then we have a function to do that :)

Reply:
Good suggestion, one which I have now implemented 👯‍♂️